tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -109,3 +116,63 @@ Verify Quick Starts Work As Expected When At Least One Step Is Skipped
     END
     Run Keyword And Continue On Failure     QuickStart Status Should Be    ${element}      In Progress
     Run Keyword And Continue On Failure     Link Text On QuickStart Card Should Be  element=${element}  exp_link_text=Continue
+
+External URLs Should Not Be Broken
+    [Documentation]     Go through a QuickStart and checks the status of all the external links
+    ${sideWindowButtons}=   Get WebElements   //button[@class='pf-c-wizard__nav-link']
+    ${element_list}=    Get WebElements    xpath=//div[@Class="pf-c-drawer__panel-main"]//a[@href]
+    @{href_list}=       Evaluate           [item.get_attribute('href') for item in $element_list]
+
+    IF  ${href_list}
+        Validate URls   ${href_list}
+    END
+    FOR    ${sideWindowButton}    IN     @{sideWindowButtons}
+            ${status}   Run Keyword And Return Status   Click Element  ${sideWindowButton}
+            IF  ${status} == False
+                Click Button    Next
+            END
+
+            FOR    ${counter}    IN RANGE    5
+                Press Keys    NONE    TAB
+            END
+
+            ${element_list}=    Get WebElements    xpath=//div[@Class="pf-c-drawer__panel-main"]//a[@href]
+            @{href_list}=       Evaluate           [item.get_attribute('href') for item in $element_list]
+            ${Doc_Text}     Get Text  //*[@class="pf-c-drawer__body pf-m-no-padding pfext-quick-start-panel-content__body"]
+            ${Doc_links}     Get Regexp Matches   ${Doc_Text}   (?:(?:(?:ftp|http)[s]*:\/\/|www\.)[^\.]+\.[^ \n]+)
```

Comment:
I think the URL validation is missing for this kind of links

Reply:
there are multiple invalid URLs in the doc  , I guess its not possible to validate it

Reply:
yeah correct working on  it